server/api/initial_data.py

```diff
@@ -821,6 +825,21 @@ def _delete_state_file():
         pass
 
 
+def _perform_version_6_data_migrations(
+    db: server.api.db.sqldb.db.SQLDB, db_session: sqlalchemy.orm.Session
+):
+    _migrate_alert_templates_table(db, db_session)
+
+
+def _migrate_alert_templates_table(
+    db: server.api.db.sqldb.db.SQLDB, db_session: sqlalchemy.orm.Session
+):
+    for template in mlrun.template_config.pre_defined_templates:
+        record = db.get_alert_template(db_session, template.template_name)
+        if record is None:
+            db.store_alert_template(db_session, template)
+
+
```

Comment:
As @quaark noted correctly, this should not happen in a data-migration context. See my comment in reply to his for where this should be.